glide.yaml

```diff
@@ -0,0 +1,19 @@
+package: github.com/giantswarm/draughtsman
+import:
+- package: github.com/giantswarm/microkit
+  version: ef2f8fdba32a0a0275d739abf84371e36880ecdc
+  subpackages:
+  - command
+  - error
+  - flag
+  - logger
+  - server
+- package: github.com/go-kit/kit
+  version: f66b0e13579bfc5a48b9e2a94b1209c107ea1f41
```

Comment:
This should not be necessary. It is a dependency of microkit. Any eventual pinning should be done there.